.github/workflows/ci.yml

```diff
@@ -35,6 +35,7 @@ jobs:
         run: |
           pip install -U pip setuptools wheel
           pip install -r dev-requirements.txt
+          pip install -r prod-requirements.txt
```

Comment:
why do you install prod-requirements?

Reply:
Thanks for the comment.
The problem occurred when the tests for some reason, executed the `__init__.py` of the `custom_storages` module, and due to `__init__.py` importing classes from `custom_storages/storages.py` and these classes depend on django-storages (that only is installed on production):

```bash
$ docker run --rm --name my_postgres -e POSTGRES_USER=postgres -e POSTGRES_PASSWORD=postgres -p 5432:5432 -d postgres:10.1
$ export DATABASE_URL=postgres://postgres:postgres@localhost:5432/pythonorg
$ python -Wd -m coverage run manage.py test -v2 
# .....
# .....
======================================================================
ERROR: custom_storages (unittest.loader._FailedTest)
----------------------------------------------------------------------
ImportError: Failed to import test module: custom_storages
Traceback (most recent call last):
  File "/home/luzfcb/.pyenv/versions/3.9.16/lib/python3.9/unittest/loader.py", line 470, in _find_test_path
    package = self._get_module_from_name(name)
  File "/home/luzfcb/.pyenv/versions/3.9.16/lib/python3.9/unittest/loader.py", line 377, in _get_module_from_name
    __import__(name)
  File "/home/luzfcb/projects/labcodes/pythondotorg/custom_storages/__init__.py", line 1, in <module>
    from custom_storages.storages import (
  File "/home/luzfcb/projects/labcodes/pythondotorg/custom_storages/storages.py", line 13, in <module>
    from storages.backends.s3boto3 import S3Boto3Storage
ModuleNotFoundError: No module named 'storages'
```

I solved it by removing any import in `__init__.py` from `custom_storages` .
Você pode, por favor, rever este pr novamente?

Reply:
> Perhaps the solution is add django-storage to dev-requirements.txt?

This really isn't necessary at this point since the project itself was never tested using production settings or production requirements.

>  How this tests run before? 

It wasn't run before because it was never a test, rather a side effect of how test autodiscover works.

> I'm asking because the django-storage is not installed, why the tests didn't fail?

The tests didn't fail before because the `custom_storages.py` module wasn't identified by the test autodiscover, and it didn't try to import it automatically.

Since the `MediaStorage` and `PipelineManifestStorage` settings only run in production, and to prevent the test autodiscover from trying to perform imports, I decided to take a step back and remove any imports from the `custom_storages/__init__.py` file and change the settings in `pydotorg/settings/heroku.py`

```diff
- DEFAULT_FILE_STORAGE = 'custom_storages.MediaStorage'
+ DEFAULT_FILE_STORAGE = 'custom_storages.storages.MediaStorage'
- STATICFILES_STORAGE = 'custom_storages.PipelineManifestStorage'
+ STATICFILES_STORAGE = 'custom_storages.storages.PipelineManifestStorage'
```

After the change, everything is back to working as expected.

Reply:
Ok, I understand!